package.json

```diff
@@ -2,25 +2,89 @@
   "name": "hill-chart",
   "description": "Basecamp's hill chart implementation in d3.js",
   "version": "1.3.1",
```

Comment:
I think we need to pump this to version 2

Reply:
Yes, I think that'd be good.